main.go

```diff
@@ -480,6 +496,23 @@ func parseSize(s string) (int64, error) {
 	return n, err
 }
 
+func getDefaultPort() (string, error) {
+	port := "/dev/ttyACM0"
```

Comment:
Not important, but it might be useful to do the same on Linux, globbing for `/dev/ttyACM*`. After removing and quickly reinserting a USB cable, the number might change.
You could do that with a switch statement:

```go
switch runtime.GOOS {
case "linux":
// ...
case "darwin":
// ...
default:
// ...
}
```

Additionally, some serial ports present as /dev/ttyUSB0 etc. but AFAIK we don't support any of those yet.

Reply:
Good idea. Done.